### PR TITLE
LPD-22076 - Attribute Name with Space: Parsed as Flag and Attribute

### PR DESCRIPTION
**Pull request by ilzamcmed**
Jira issue: https://liferay.atlassian.net/browse/LPD-22076

**Issues**

- If we add an attribute name with a space between the words, the first word is added in the script as a flag, and the second word as the real attribute;

### CODE REVIEW

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

File: modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/global-js/AttributeFields.tsx [hunk 1]
```diff
@@ -64,10 +64,19 @@ export default function AttributeFields({
 					disabled={disabled}
 					errorMessage={errorMessage}
 					id={nameId}
-					label={Liferay.Language.get('attribute')}
+					label={
+						<>
+							{Liferay.Language.get('attribute')}
+
+							<span className="sr-only">
+								Spaces are not allowed.
```
Comment (ilzamcmed) [inline modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/global-js/AttributeFields.tsx:72]: I forgot to add the language key here. Waiting for review to send the fix.

**Comment by ilzamcmed** [comment]
The relevant failures are unrelated to this change.

**Comment by ethib137** [comment]
Thanks Ilza! Let’s review this on a call on Monday.

**Comment by ilzamcmed** [comment]
c:test:sf

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
Investigating SF failures

**Comment by ilzamcmed** [comment]
ci:test:sf

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Comment by ilzamcmed** [comment]
ci:stop

**Comment by ilzamcmed** [comment]
ci:test:relevant

**Review by ethib137 — APPROVED** [review]
LGTM... If the tests pass feel free to forward.

**Comment by ilzamcmed** [comment]
Relevant failures are unrelated.

**Comment by ilzamcmed** [comment]
ci:forward:force

**Pull request closed without merging**